### DISCUSSION TITLE
Shapes get filled in

### DISCUSSION THREAD

**Discussion by SanelyInproductive** [xournalpp/xournalpp#3055] [Q&A] [answered]
Hello;
when I draw something the forms get filled in although I don't want that. 
If for example I am drawing a circle I get this 
![image](https://user-images.githubusercontent.com/79225950/115384829-c4662080-a1d7-11eb-96a4-5dd9f895cea4.png) 
instead of 
![image](https://user-images.githubusercontent.com/79225950/115384904-d942b400-a1d7-11eb-8e67-d2aa7b15775b.png)

Could anyone please tell me how to fix this.
Thanks in advance!

**Comment by rolandlo** [comment, marked as answer]
You can switch between filled and not filled using the fill bucket
![grafik](https://user-images.githubusercontent.com/40485433/115390289-5e30cc00-a1de-11eb-94f9-075a7041c442.png)
or alternatively via menu `Tools > Pen Options`

**Reply by SanelyInproductive** [reply]
Thank you very much
